src/tab-active.js

```diff
@@ -1,19 +1,20 @@
-function wikiaJWPlayerHandleTabNotActive(playerInstance) {
-	function canPlayVideo() {
-		return !document.hidden && (['playing', 'paused', 'complete'].indexOf(
-			playerInstance.getState()) === -1 || pausedOnRelated);
-	}
-
+function wikiaJWPlayerHandleTabNotActive(playerInstance, willAutoplay) {
 	var pausedOnRelated = false;
 
-	playerInstance.on('playerStart', function () {
-		document.addEventListener('visibilitychange', function () {
-			if (canPlayVideo()) {
+	function playVideoOnTabSwitch() {
+		if (['playing', 'paused', 'complete'].indexOf(playerInstance.getState()) === -1 || pausedOnRelated) {
```

Comment:
Don't we need `document.hidden` check anymore? I looks like it can start play now when someone is leaving the tab.